src/fem/discrete_space.hh

```diff
@@ -0,0 +1,111 @@
+/*!
+ *
+﻿ * Copyright (C) 2015 Technical University of Liberec.  All rights reserved.
+ * 
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 3 as published by the
+ * Free Software Foundation. (http://www.gnu.org/licenses/gpl-3.0.en.html)
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ * 
+ * @file    dofhandler.hh
```

Comment:
Update file header.

Reply:
Not clear what is the purpose of DiscreteSpace class. Just wrap FE for elements of different dimension? For p-refined or XFEM case it could be better to have elementwise additional data as part of the DofHandler itself.